### ISSUE TITLE
training loss does not go down

### ISSUE THREAD

**Issue by 363325971** [lengstrom/fast-style-transfer#178]
I trained a model for 20 hours, but the loss did't decrease, anyone could help?

**Comment by JEF1056** [comment]
What are your computer's specs?
